src/app/app.component.html

```diff
@@ -300,11 +300,11 @@
                       </li>
                       <li class="nav-item">
                         <a class="nav-link" [ngClass]="{'active': activeTab === 'HEADERS'}"
-                        (click)="setActiveTab('HEADERS')">Headers</a>
+                        (click)="setActiveTab('HEADERS')">Headers{{(activeRouteResponseHeaderCount$ | async) > 0 ? ' (' + (activeRouteResponseHeaderCount$ | async)+')' : ''}}</a>
```

Comment:
Could you please subscribe only once by using an `*ngIf` and `as`:
```
<a *ngIf="activeRouteResponseHeaderCount$ | async as activeRouteResponseHeaderCount" ....>
```

Reply:
Thanks for the tip! TIL 😀

Reply:
Whoops, totally forgot about it!

I tried to implement it just now, but as far as I can see the `*ngIf` hides the `<a />` when rule/header count equals 0.
If I create 2 separate anchor tags, one for when rules exist and one for when they don't it will end up with two subscriptions again.
Can you think of a workaround to that?

Reply:
I think you could use `ng-container` like this:
```
<a>
Header<ng-container *ngIf="obs$ | async as obs"> ({{obs}})</ng-container>
</a>
```